rpmrc.in

```diff
@@ -243,6 +247,9 @@ arch_canon:	mipsr6el: mipsr6el	20
 arch_canon:	mips64r6: mips64r6	21
 arch_canon:	mips64r6el: mips64r6el	21
 
+arch_canon:	riscv: riscv64	22
```

Comment:
The rest of the file uses single tabs between fields, and as a result doesn't line up either.